sdk/highlight-remix/package.json

```diff
@@ -0,0 +1,50 @@
+{
+	"name": "@highlight-run/remix",
+	"version": "0.0.0",
+	"description": "Client for interfacing with Highlight in Remix",
+	"packageManager": "yarn@3.5.0",
+	"author": "",
+	"license": "ISC",
+	"files": [
+		"dist",
+		"highlight-init.d.ts",
+		"handle-error.d.ts",
+		"index.d.ts"
+	],
+	"exports": {
+		"./server": {
+			"types": "./dist/server.d.ts",
+			"import": "./dist/server.mjs",
+			"require": "./dist/server.js"
+		},
+		"./client": {
+			"types": "./dist/client.d.ts",
+			"import": "./dist/client.mjs",
+			"require": "./dist/client.js"
+		},
+		"./report-dialog": {
+			"types": "./dist/report-dialog.d.ts",
+			"import": "./dist/report-dialog.mjs",
+			"require": "./dist/report-dialog.js"
+		}
+	},
+	"scripts": {
+		"typegen": "tsup --dts-only",
+		"dev": "tsup --watch",
+		"build": "tsup"
+	},
+	"peerDependencies": {
+		"highlight.run": ">=3.0.0",
```

Comment:
doesn't this need to be `>=7.4.0` to get the new `init` return behavior?

Reply:
Yep! Fixing.